kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectFromBookmarks.vue

```diff
@@ -79,7 +79,11 @@
 
       const { data, hasMore, fetchMore, loadingMore } = props.bookmarksFetch;
 
-      const contentCardMessage = content => {
+      const wrappedContentCardMessage = content => {
```

Comment:
This is maybe a nitpick, but I really don't understand what the `wrapped` is conveying here.

Reply:
It's because `contentCardMessage` is already the name of a prop, so it gives a name collision if I exported it with the same name.

As it is wrapping the `contentCardMessage` prop (it invokes it first), I thought `wrappedContentCardMessage` was as good a name as any.

Reply:
fair enough!